docs/docs/user-guide/training-data-format.mdx

```diff
@@ -0,0 +1,659 @@
+the NLU pipeline. In the example above, the sentiment of 
+the example is recorded as metadata.
+
+### Entities
+
+Where applicable, [entities])(../glossary#entity) can be annotated in training examples using either the
+abbreviated syntax:
+
+```yaml
+[<entity-text>]("<entity name>")
+```
+
+Or the syntax:
+
+```yaml
+[<entity-text>]{"entity": "<entity name>"}
+```
+
+
+In a training example, this would look like:
+
+```yaml
+nlu:
+- intent: check_balance
+  examples: |
+  - how much do I have on my [savings]("account") account
+  - how much money is in my [checking]{"entity": "account"} account
+  - What's the balance on my [credit card account]{"entity":"account","value":"credit"}
+
+```
+
+The third example above includes a [synonym](#synonyms).
+You can assign synonyms, roles, or groups to an entity using the
+syntax:
+
+```yaml
+[<entity-text>]{"entity": "<entity name>", "role": "<role name>", "group": "<group name>", "value": "<entity synonym>"}
+```
+
+The keywords `role`, `group`, and `value` are optional in this notation.
+The `value` keyword refers to synonyms, which are explained in the
+following section. To understand what the labels `role` and `group` are
+for, see the section on (entity roles and groups)[nlu/entity-extraction#entities-roles-and-groups].
+
+### Synonyms
+
+Synonyms provide a way to normalize your training data by mapping an
+extracted entity to a value other than the literal text extracted.
+Synonyms can be defined in the format:
+
+```yaml
+- synonym: <synonym value>
+  examples: |
+  - <a synonym variation>
+  - <another synonym variation>
+```
+
+Synonyms can also be definedin-line in your training examples by
+specifying the `value` of the entity:
+
+```yaml
+nlu:
+- intent: check_balance
+  examples: |
+  - how much do I have on my [credit card account]{"entity": "account", "value": "credit"}
+  - how much do I owe on my [credit account]{"entity": "account", "value": "credit"}
+```
+
+To use the synonyms defined in your training data, you need to make sure the
+pipeline contains the [`EntitySynonymMapper`](../nlu/components#entitysynonymmapper) component. You
+should define synonyms when there are multiple ways users refer to the same
+thing.
+
+#### Example 
+
+For example, let's say you had an entity `account`, and you expect the
+value "credit". Your users also refer to their "credit" account as "credit
+account" and "credit card account".
+
+In this case, you could define "credit card account" and "credit account" as
+**synonyms** to "credit":
+
+```yaml
+- synonym: credit
+  examples: |
+  - credit card account
+  - credit account
+```
+
+Then, if either of these phrases is extracted as an entity, it will be
+mapped to the **value** `credit`.
+
+:::note
+Synonym mapping only happens **after** entities have been extracted.
+That means that you need to provide examples of the variations 
+on a synonym in your training examples so
+that Rasa Open Source can learn to pick them up.
+:::
```

Comment:
How about following the example: 

Synonym mapping only happens **after** entities have been extracted.
That means that your training examples should include the synonym examples (`credit card account` and `credit account`) so that the model will learn to recognize these as entities and replace them with `credit`.